src/App.tsx

```diff
@@ -1,57 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { Dropdown } from './components/Dropdown';
+
+const SEARCH_DELAY = 1000;
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
 
   return (
     <main className="section">
-      <h1 className="title">
-        {`${name} (${born} = ${died})`}
+      <h1 className={classNames({ title: selectedPerson })}>
+        {selectedPerson
+          ? (
+            `${selectedPerson.name} (${selectedPerson.born} = ${selectedPerson.died})`
```

Comment:
```suggestion
            `${selectedPerson.name} (${selectedPerson.born} - ${selectedPerson.died})`
```
I think it should be `-` instead of `=`
![image](https://github.com/mate-academy/react_autocomplete/assets/63243136/868f16fc-06e5-4b77-a2e9-de0f3ac3053c)